### DISCUSSION TITLE
Eleventy Edge using 11ty.js templates

### DISCUSSION THREAD

**Discussion by lttr** [11ty/eleventy#2357] [Q&A]
I'm having troubles with how to make Eleventy edge working with 11ty.js templates. The process is a little bit too heavy (edge handling inside liquid template inside javascript template inside my site setup).

There might be an issue with the docs right now: On page https://www.11ty.dev/docs/plugins/edge/#5.-make-your-content-template
there is only `{% endedge %}` in the 11ty.js tab, but no `{% edge %}`. Is that correct?

I would also appreciate some tips for debugging this. Using `console.log` or following errors of the `netlify dev` server is not helping me this time.

**Comment by zachleat** [comment]
Not at the computer at the moment but that stray endedge tag should not be on the 11ty.js example
